internal/engine/table.go

```diff
@@ -43,16 +41,9 @@ type Inserter interface {
 type Table struct {
 	profiler *profile.Profiler
 
-	// storage-facing fields
-
-	tablePage PageContainer
-	dataPage  PageContainer
-
-	// application-facing fields
-
 	name         string
-	highestRowID uint32
-	cols         []table.Col
+	schema       *schema.Schema
```

Comment:
Is `schema` a right name for this? I understand that schema means any plan but in the future of this db being used by people I dont want someone to get lost thinking this is the DB schema.

Reply:
> A relation schema is the logical definition of a table - it defines what the name of the table is, and what the name and type of each column is. It's like a plan or a blueprint. A database schema is the collection of relation schemas for a whole database.

I also don't like the ambiguity. I also thought about putting it into table, but for now I wanna keep it as `schema.Schema`.

Reply:
Sure. Pin this or write something about this as a comment maybe for the future.

Reply:
It's in the godoc to the `Schema` struct